### DISCUSSION TITLE
Netmaker + AdGuard + Unbound

### DISCUSSION THREAD

**Discussion by dikiyvaper228** [gravitl/netmaker#2272] [General]
It's been a few days now and I can't get it to work. I install and configure adguard and unbound according to the guide from https://github.com/trinib/AdGuard-WireGuard-Unbound-DNScrypt. But when setting up the config in netmaker, it shows my own IP address. I tried to change the network settings in the netmaker ui and nothing changes

**Comment by miyu4u** [comment]
You need to set up an egress gateway on the netmaker server node that can communicate with the device where adguard home is installed.
